webapp/src/components/viewHeader/emptyCardButton.scss

```diff
@@ -0,0 +1,22 @@
+@-moz-document url-prefix('') {
```

Comment:
It looks like this file change is duplicated from #2849, was that the intention?

Reply:
My bad, I have to checkout this branch from the branch I was working on rather than the main, I will make the changes and push it. Thank you.